CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Two functions (`shadow.to-box-shadow()` and `shadow.to-drop-shadow()`) for translating those shadows (which are specified in Sass' list format) into either CSS `box-shadow` format, or CSS filter `drop-shadow` format. Not necessary for the shadows accessed using `shadows.get()`
 - New `u-shadow` and `u-drop-shadow` utility classes to set box-shadows and drop-shadows respectively
 - The smallscreen version of the Sidebar organism can now be positioned to the left or right side of the screen by overriding `$bitstyles-sidebar-small-location` (defaults to `left`)
+- New atom component `a-skip-link`, for skip-to-content links
```

Comment:
I did not know of this concept 💡 

Thanks! It's [this](https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G1), right?

Why is it called `skip-link` and not `skip-to-content`? Is this the general term for it?

Reply:
Exactly — it’s  just shorter, it’s become the common term for it